context/app/static/js/components/App.jsx

```diff
@@ -1,20 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Container from '@material-ui/core/Container';
 import Providers from './Providers';
 import Routes from './Routes';
 import Footer from './Footer';
-import Header from './Header';
+import { Header } from './Header';
```

Comment:
I am following a pattern of adding a domain level `index.js` to export components used elsewhere in the application. Since some of these index files may include multiple exports I thought it made sense to use named exports throughout for consistency versus having some defaults and some named.